packages/js/product-editor/src/blocks/checkbox/README.md

```diff
@@ -0,0 +1,45 @@
+# woocommerce/product-checkbox-field
+
+A reusable checkbox for the product editor.
+
+_Please note that to persist a custom field in the product it also needs to be added to the WooCommerce REST API._
+
+| **attribute** | **type** | **required** | **description**                                                            |
+|--------------:|----------|--------------|----------------------------------------------------------------------------|
+| title         | string   | no           | Header that appears above the checkbox                                     |
+| label         | string   | no           | Label that appears at the side of the checkbox                             |
+| property      | string   | yes          | Property in which the checkbox value is stored                             |
+| tooltip       | string   | no           | Tooltip text that is shown when hovering the icon at the side of the label |
```

Comment:
I don't mind this formatting. Actually, it's kinda nice. It does differ more from Gutenberg's formatting (see https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-card/README.md for one example) and the (very similar to Gutenberg) formatting that I went with in my recent PR (https://github.com/woocommerce/woocommerce/pull/40263 -- see the README.md files).

I think I ultimately prefer a non-table format, as I think it offers more flexibility in providing longer descriptions for the attributes.